tests/slack_sdk/models/test_elements.py

```diff
@@ -129,6 +129,12 @@ def test_text_length(self):
                 text=STRING_301_CHARS, action_id="button", value="click_me"
             ).to_dict()
 
+    def test_action_id_length(self):
```

Comment:
This is implemented in its super class. Just in case, I've added more tests for it.